### PR TITLE
Various enhancements

### PR DESCRIPTION
**Pull request by robertkiel**
Re https://github.com/hoprnet/hoprnet/issues/5951

# Invalid links

Setting the `mainService` property in `dappnode_package.json` assigns the main DNS entry, in this case `hopr.public.dappnode` to the defined service, which in our case is `node`.

This PR changes all URLs directing to `node.hopr.public.dappnode` to `hopr.public.dappnode`, including the prometheus target

# Yarn removal

The `dappnodesdk` uses `npm` and `npx` (`npx @dappnode/dappnodesdk build`) and refuses to work with `yarn ` for unknown reasons. This PR removes all `yarn`-related settings and files.

# Enhance development documentation

This PR adds helpful information on how to build and debug DAppnode packages. See DEVELOPMENT.md

# Out of scope

- Automatic import of prebuild Grafana dashboards as it appears to be broken at the moment.
- Automatic retrieval of IPFS hash when releasing package
- Automatic patching of Grafana dashboard (take the one used in Kubernetes and adapt it for DAppnode) by replacing incorrect entries with `jq`

### CODE REVIEW

**Review by NumberFour8 — APPROVED** [review]

File: DEVELOPING.md [hunk 1]
```diff
@@ -16,5 +15,49 @@ Tha development happens on the internal repository and it has two branches:
...
+
+```sh
+npx @dappnode/dappnodesdk build
+```
+
+This should create output similar to
+
+```
+  ✔ Verify connection
+  ✔ Verify connection
```
Comment (Teebor-Choka) [inline DEVELOPING.md:36]: Copy paste?

File: dappnode_package.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
   ],
   "links": {
     "homepage": "https://hoprnet.org/",
-    "ui": "http://admin.hopr.public.dappnode/?apiEndpoint=http://node.hopr.public.dappnode:3001",
-    "api": "http://node.hopr.public.dappnode:3001",
+    "ui": "http://admin.hopr.public.dappnode/?apiEndpoint=http://hopr.public.dappnode:3001",
+    "api": "http://hopr.public.dappnode:3001/swagger-ui/index.html",
```
Comment (Teebor-Choka) [inline dappnode_package.json:24]: Why would the API point to swagger?

Comment (robertkiel) [reply dappnode_package.json:24]: It's just an entry in the UI

File: DEVELOPING.md (hunk 1 above)
Comment (robertkiel) [reply DEVELOPING.md:36]: No, original output from `dappnodesk`

**Pull request merged**